packages/core/src/Modules/Cashier/Components/confirm.jsx

```diff
@@ -16,7 +17,13 @@ const Row = ({ label, value }) => (
                 ))}
             </span>
         ) : (
-            <strong>{value}</strong>
+            <strong
+                className={classNames({
+                    desc: key === 'description',
```

Comment:
can we follow  the`bem` pattern for the classname?

Reply:
It makes selecting the element a bit hard and complex. Let's keep it as is.